input/gcp/gcp.go

```diff
@@ -58,7 +58,9 @@ func configure(cfg *conf.C) (stateless.Input, error) {
 
 func newAssetsGCP(config config) (*assetsGCP, error) {
 	vpcAssetsCache, _ := freelru.New[string, *vpc](8192, hashStringXXHASH)
-	return &assetsGCP{config, vpcAssetsCache}, nil
+	subnetAssetsCache, _ := freelru.New[string, *subnet](8192, hashStringXXHASH)
+	computeAssetsCache, _ := freelru.New[string, *computeInstance](8192, hashStringXXHASH)
```

Comment:
NIT: We could use the new `getEmpty....` functions that you defined in `util.go` here

Reply:
I thought about it but I didn't do it in the end. Will do